agreement/asyncVoteVerifier.go

```diff
@@ -76,19 +77,19 @@ func MakeAsyncVoteVerifier(verificationPool execpool.BacklogPool) *AsyncVoteVeri
 
 	verifier.ctx, verifier.ctxCancel = context.WithCancel(context.Background())
 
-	verifier.wg.Add(1)
+	verifier.workerWaitCh = make(chan struct{})
 	go verifier.worker()
 	return verifier
 }
 
 func (avv *AsyncVoteVerifier) worker() {
-	defer avv.wg.Done()
+	defer func() { close(avv.workerWaitCh) }()
```

Comment:
Why do you need to wrap in func()?  Won't `defer close(avv.workerWaitCh)` do the same thing?

Reply:
yes; updated.